helper_crates/vtable/src/vrc.rs

```diff
@@ -126,6 +128,12 @@ impl<VTable: VTableMetaDropInPlace + 'static, X> Drop for VRc<VTable, X> {
     }
 }
 
+impl<VTable: VTableMetaDropInPlace + 'static, X> core::fmt::Debug for VRc<VTable, X> {
```

Comment:
So it means we cannot implement a version when `X : Debug`  that prints the content of the pointer, like std::rc::Rc does.

I guess that's alright.